pkg/kubernetes/app.js

```diff
@@ -118,6 +152,16 @@ define([
             };
         })
 
+        .filter("namespaceDisplay", function() {
```

Comment:
Lets not yet use these annotations. I was talking with the UX guys at the container UI meeting about this. Use of the annotation display names conflicts with our goal of being able to interchange using the command line and the Cockpit UI.